server/repository/notificationRepository.js

```diff
@@ -0,0 +1,40 @@
+import Model from '../models';
+import notifyWorker from '../services/notifyWorker';
+
+const { Notifications } = Model;
+
+/**
+ * Tag repository class
+ */
+class NotificationRepository {
+  /**
+ * Function to create a new notification
+ * @param {object} notification
+ * @returns {object} notification
+ */
+  static async createNotification(notification) {
+    let content;
+    let newNotification = {};
+    switch (notification.type) {
+      case 'NEW_ARTICLE_UPDATE':
```

Comment:
These should be constants exported into the file `NEW_ARTICLE_UPDATE `

Reply:
Noted and fixed.